main.rb

```diff
@@ -0,0 +1,31 @@
+require './app'
+
+def list_options
+  options = ['Please choose an option by entering a number', '1 - List all books', '2 - List all people',
+             '3 - Create a person', '4 - Create a book', '5 - Create a rental',
+             '6 - List all rentals for a given person id', '7 - Exit']
+  puts
+  options.each { |option| puts option }
+end
+
+def parameter_option(parameter)
+  options = {
+    '1' => method(:list_books),
+    '2' => method(:list_people),
+    '3' => method(:option_create_person),
+    '4' => method(:option_create_book),
+    '5' => method(:option_create_rental),
+    '6' => method(:option_list_rentals),
+    '7' => method(:exit)
+  }
+
+  option_method = options[parameter] || method(:invalid_option)
+  option_method.call
+end
+
+def main
+  app = App.new
+  app.begin_console
+end
+
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** I think it would be better if you add a command of how to run your application in the `Usage` sub-section to make your documentation more descriptive and professional. i.e.
```
### Usage

To run the project, execute the following command:

ruby main.rb

```